src/Authentication/Authenticators/Session.php

```diff
@@ -520,6 +529,15 @@ public function isPending(): bool
         return $this->userState === self::STATE_PENDING;
     }
 
+    /**
+     * Checks if the user is currently banned.
+     * Their account needs to be unbanned.
+     */
+    public function isBanned(): bool
+    {
+        return $this->getUser()->banned;
```

Comment:
This should fix the error during tests. 

```suggestion
        return (bool)$this->getUser()->banned;
```